orekit/src/utils.py

```diff
@@ -0,0 +1,57 @@
+""" Utilities for working with Orekit wrapper """
+
+import logging
+import os
+
+import requests
+from orekit.pyhelpers import setup_orekit_curdir
+
+import orekit
+
+OREKIT_SRC_DIR = os.path.dirname(os.path.abspath(__file__))
+OREKIT_DATA_DIR = os.path.join(OREKIT_SRC_DIR, "data")
+OREKIT_DATA_PATH = os.path.join(OREKIT_DATA_DIR, "orekit-data.zip")
+
+
+def build_data_dir(dirname_path=OREKIT_DATA_DIR):
+    """ Builds orekit data directory """
+
+    try:
+        os.mkdir(dirname_path)
+    except FileExistsError:
+        pass
+
+
+def download_orekit_data(save_path=OREKIT_DATA_PATH, chunk_size=128):
+    """ Downloads orekit ZIP data """
+
+    # Prepare the logger
+    log = logging.getLogger(__name__)
```

Comment:
Sorry, I should have explained myself better! The instantiation should happen at module level. I'll push the change to your branch and merge.